docs/library/machine.SDCard.rst

```diff
@@ -50,6 +50,8 @@ vary from platform to platform.
      - *mosi* can be used to specify an SPI mosi pin.
 
      - *cs* can be used to specify an SPI chip select pin.
+     
+     - *freq_khz* selects the SD/MMC interface frequency in KHz unit.
```

Comment:
Shouldn't you indicate that this arg is esp32-only?

Reply:
Is there a separtate doc section about esp32 SDcard class?

Reply:
I don't think so. I would just add a "Note: only supported on the ESP32" or something like that...

Reply:
OK